extensions/wikia/UserLogin/templates/UserLoginSpecial_Providers.php

```diff
@@ -1,4 +1,4 @@
-	<div class="sso-login">
+	<div class="sso-login fb-loaded hidden">
```

Comment:
This class seems like a state marker, and by default (i.e. when this html is loaded), facebook is not yet loaded. Is there any reason we can't use `.sso-login`? That element might even be already cached in some cases.